user-guide/rules.md

```diff
@@ -463,67 +466,94 @@ Apart from returning files or predefined responses, a rule can perform the follo
+The **Rules Queue** is a visual tree that lists all active rules & groups in a priority queue. It is important to note that each active rule or group has higher execution priority than other rules & groups placed lower in the queue. The **Rules Queue** allows you to enable, disable, promote, denote, edit, export, import, share, and immediately execute the listed rules.
 
-When using a rule from the **Rules Queue**, take into consideration the following tips:
+To use a rule (or a group of rules) from the **Rules Queue**, take into consideration the following tips:
 
-- The **Rules** tab must be enabled for any rule from the queue to be executed.
-- Only **enabled** rules from the queue will be applied on ongoing Live traffic capture.
-- You can explicitly apply a rule for captured traffic by using the **Play** button (executes the rule on current sessions).
-- The rules in the **Rules Queue** are executed with priority based on their position in the queue. Use **Demote** and **Promote** buttons to change a rule position (and priority) in the queue.
-- Some rules might trigger [a final action](#final-and-non-final-actions), which will stop executing any other rules with lower priority.
+- The **Rules** tab must be active (switch ON).
 
-### Rules Priority
+- Only **active** rules from the queue will be applied on ongoing Live traffic capture. If the rule is contained within a group, then the group must be active as well.
+
+- You can explicitly trigger a rule for already captured (including saved) traffic through the **Execute** button (executes the rule on current sessions).
+
+- The **Execute** button is not affected by the active state of groups/rules. Actions from any selected rule or group of rules will be triggered when the **Execute** button is used.
+
+- The rules in the **Rules Queue** are executed with priority based on their position in the queue. Use the **Demote** and **Promote** buttons to change the queue's rule position (and priority). Alternatively, you can drag and drop rules to change their position in the queue. 
 
-By default, the **Rules Queue** will list all created and imported rules based on their priority. Rules will be executed according to their appearance, starting from the first. When selecting specific rules and using the explicit **Execute** button, they will run in the order they appear in the queue.
+- The groups in the **Rules Queue** are executed with priority based on their position in the queue. Use the **Demote** and **Promote** buttons to change the queue's group position (and priority). Alternatively, you can drag and drop groups to change their position in the queue.
 
-### Automatic Rules Execution for Live Traffic
+- You can use drag and drop rules in and out of groups and subgroups. Note that this might change the execution priority when multiple rules are executed simultaneously.
```

Comment:
```suggestion
- You can drag and drop rules in and out of groups and subgroups. Note that this might change the execution priority when multiple rules are executed simultaneously.
```